prime-router/settings/prod/0027-la-fix.yml

```diff
@@ -0,0 +1,43 @@
+# Run this with ./prime receiver set --env prod --input ./settings/prod/0027-la-fix.yml
```

Comment:
This was an emergency fix - already done.   There was a missing comma in the orEquals filter.